package.json

```diff
@@ -48,5 +48,9 @@
     "testPathIgnorePatterns": [
       "<rootDir>/src"
     ]
+  },
+  "volta": {
+    "node": "12.22.12",
```

Comment:
Will be out-of-sync once we merge https://github.com/getsentry/sentry-cli/issues/1391 so lets wait for `#1391`.